### ISSUE TITLE
Missing generated files for Lazy Load demo

### ISSUE THREAD

**Issue by Ne-Ne** [BBC-News/Imager.js#133]
I had to create the images for the lazy load demo to work.

Is this intentional?

**Comment by nealoke** [comment]
@Ne-Ne yes ofcourse :) This is just to select the correct width / height based on the viewport
